op.c

```diff
@@ -8018,6 +8018,15 @@ Perl_utilize(pTHX_ int aver, I32 floor, OP *version, OP *idop, OP *arg)
 
         U16 shortver = S_extract_shortver(aTHX_ use_version);
 
+        if (shortver && PL_prevailing_version) {
+            /* use VERSION while another use VERSION is in scope
+             * This should provoke at least a warning, if not an outright error
+             */
+            /* downgrading from >= 5.11 to < 5.11 is now fatal */
+            if (PL_prevailing_version >= SHORTVER(5, 11) && shortver < SHORTVER(5, 11))
+                croak("Downgrading a use VERSION declaration to below v5.11 is not permitted");
+        }
+
```

Comment:
I should add, as a pre-review comment, that the reason for this slightly-odd code structure is that I have some more commits that add other warnings or errors in this same section. So it's a weird shape at the moment but it'll make more sense later.